packages/components/src/draggable/index.tsx

```diff
@@ -212,10 +212,6 @@ export function Draggable( {
 		// Update cursor to 'grabbing', document wide.
 		ownerDocument.body.classList.add( bodyClass );
 
-		// Allow the Synthetic Event to be accessed from asynchronous code.
-		// https://reactjs.org/docs/events.html#event-pooling
-		event.persist();
```

Comment:
Apologies in case this has already been discussed (I couldn't find a related convo in this PR).

What is the reason for removing this line of code?

Reply:
Thank you for the context! I think it's fine to keep it in this PR, as far as we add a little CHANGELOG entry to leave around some context for the change :)

Reply:
Added changelog in [8bbd1a2](https://github.com/WordPress/gutenberg/pull/49673/commits/8bbd1a25057a00fd5cbb28e7647f6f7354b8d5ff).

Reply:
Oh, oops! I added it in the wrong place. Updated it in [f1cd8a3](https://github.com/WordPress/gutenberg/pull/49673/commits/f1cd8a3534ad33ea32db972c59944e70bbca024f).